lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.html

```diff
@@ -29,5 +29,10 @@
                     <ng-content select="adf-custom-empty-content"></ng-content>
                 </ng-template>
             </adf-no-content-template>
+
```

Comment:
Why do we have this commented code?

Reply:
It was used to test the json cell with data-columns. I will remove that.